apps/versions/tests.py

```diff
@@ -94,6 +94,21 @@ def test_dict_from_int():
     eq_(d['pre_ver'], 2)
 
 
+@pytest.mark.parametrize("addon_type", amo.GROUP_TYPE_ADDON)
```

Comment:
Eh, it's been created by a german, then maintained by many "non english native speakers" ;)

For reference:
- http://pytest.org/latest/parametrize.html#parametrized-test-functions
- http://pytest.org/latest/fixture.html#fixture